WordPress/UITests/README.md

```diff
@@ -85,17 +85,44 @@ Note that due to the mock server setup, tests cannot be run on physical devices
 
 When adding a new UI test, consider:
 
-* Whether you need to test a user flow (to accomplish a task or goal) or a specific feature (e.g. boundary testing).
-* What screens are being tested (defined as page objects in `Screens/`).
-* Whether there are repeated flows across tests (defined in `Flows/`).
-* What network requests are made during the test (defined in `API-Mocks/`).
+* Whether to test a user flow (to accomplish a task or goal) or a specific feature (e.g. boundary testing).
+* What screens are being tested (defined as screen objects in [.../Screens](https://github.com/wordpress-mobile/WordPress-iOS/tree/trunk/WordPress/UITestsFoundation/Screens)).
+* Whether there are repeated flows across tests (defined in [.../Flows](https://github.com/wordpress-mobile/WordPress-iOS/tree/trunk/WordPress/UITests/Flows)).
+* What network requests are made during the test (defined in [.../API-Mocks](https://github.com/wordpress-mobile/WordPress-iOS/tree/trunk/API-Mocks)).
 
-It's preferred to focus UI tests on entire user flows, and group tests with related flows or goals in the same test suite.
+Tests classes are grouped together in [.../Tests](https://github.com/wordpress-mobile/WordPress-iOS/tree/trunk/WordPress/UITests/Tests)
```

Comment:
I'm not familiar with the `.../` syntax, is that a standard way to say "a older defined somewhere"?

Reply:
oh that was to say that it's in that folder, would just `/Tests` be clearer?

Reply:
I'd vote for just using the folder name, w/o `...` or `/`, to avoid making it look like a relative reference.